src/main/java/net/glowstone/GlowServer.java

```diff
@@ -931,7 +932,7 @@ private void enablePlugins(PluginLoadOrder type) {
             // Default permissions
             this.permissionRoot = DefaultPermissions.registerPermission("minecraft", "Gives the user the ability to use all Minecraft utilities and commands");
             this.permissionCommand = DefaultPermissions.registerPermission("minecraft.command", "Gives the user the ability to use all Minecraft commands", permissionRoot);
-            DefaultPermissions.registerPermission("minecraft.command.say", "Allows the user to speak using a command", PermissionDefault.TRUE, permissionCommand);
```

Comment:
Oh hm, I guess I was thinking of a plugin that replaced `say`, which only sent a chat message like you would normally, instead of broadcasting the message.